udata_recommendations/schema.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
A basic JSON schema to validate JSON used as inputs. It's pretty laxist, I'm open to additions